Networking/BLIP/BLIPConnection.cc

```diff
@@ -45,8 +45,8 @@ namespace litecore { namespace blip {
     const char* const kMessageTypeNames[8] = {"REQ", "RES", "ERR", "?3?",
                                               "ACKREQ", "AKRES", "?6?", "?7?"};
 
-    LogDomain BLIPLog("BLIP", LogLevel::Debug);
-    static LogDomain BLIPMessagesLog("BLIPMessages", LogLevel::Debug);
+    LogDomain BLIPLog("BLIP", LogLevel::Warning);
+    static LogDomain BLIPMessagesLog("BLIPMessages", LogLevel::None);
```

Comment:
Is this changeable via CBL API?  If not, then what good is this domain doing?

Reply:
I don't know the original intent, but I think it's useful for Core debugging with Xcode.

Reply:
All I am here doing is to restore their values to the values before I changed them to catch more infos for the flaky failures I could only hope to see in Jenkins build.